<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buildingOfFloor</class>
 <widget class="QWidget" name="buildingOfFloor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>楼层图</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_93">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>楼层按钮</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_115">
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up20">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down20">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_112">
         <property name="text">
          <string>19</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up19">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down19">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_111">
         <property name="text">
          <string>18</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up18">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down18">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_114">
         <property name="text">
          <string>17</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up17">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down17">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_113">
         <property name="text">
          <string>16</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up16">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down16">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_96">
         <property name="text">
          <string>15</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up15">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down15">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_97">
         <property name="text">
          <string>14</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up14">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down14">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_98">
         <property name="text">
          <string>13</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up13">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down13">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_99">
         <property name="text">
          <string>12</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up12">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down12">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_101">
         <property name="text">
          <string>11</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up11">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down11">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_103">
         <property name="text">
          <string>10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up10">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down10">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_102">
         <property name="text">
          <string>9</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up9">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down9">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_105">
         <property name="text">
          <string>8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up8">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down8">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_107">
         <property name="text">
          <string>7</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up7">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down7">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_100">
         <property name="text">
          <string>6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up6">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down6">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_110">
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up5">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down5">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_109">
         <property name="text">
          <string>4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up4">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down4">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_108">
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up3">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down3">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_106">
         <property name="text">
          <string>2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up2">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down2">
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_95">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up1">
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="down1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
